app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
The `sorted_variables` dictionary is a bit confusing. The keys 'mutable' and 'immutable' suggest that the lists contain mutable and immutable types, but this is not entirely accurate. For example, the `profile_info` tuple contains strings, which are immutable. Consider revising this part of the code to improve clarity.